<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QDialog" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5" rowstretch="1,6,6">
   <property name="bottomMargin">
    <number>80</number>
   </property>
   <item row="0" column="0">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="softerwareVersion">
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;
font: 63 18pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>软件版本：M16.S92.M2718.AHD</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="languageLabel">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>72</width>
           <height>24</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;
font: 63 18pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>语言：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="languageSelect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>41</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>中文简体</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>English</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="typeLabel2">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;
font: 63 16pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>型号：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="typeLabel1">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;
font: 63 20pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>T3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>111</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 14pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>SD卡状态</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QProgressBar" name="ROM_usage">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="formatButton">
       <property name="minimumSize">
        <size>
         <width>91</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>91</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>格式化</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
